{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf830
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
{\*\expandedcolortbl;;}
\margl1440\margr1440\vieww10800\viewh8400\viewkind0
\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0

\f0\fs24 \cf0 Run \'91CLASSIFICATION-RUNME.py\'92 to classify the images.\
\
Please note, before you can run this you must download the AlexNet weights \'92bvlc_alexnet.npy\'92 from {\field{\*\fldinst{HYPERLINK "http://www.cs.toronto.edu/~guerzhoy/tf_alexnet/"}}{\fldrslt http://www.cs.toronto.edu/~guerzhoy/tf_alexnet/}} and 
\b place this file into the \'91code\'92 folder
\b0  in order for the classification to work. It would be included if not for file size limits on GitHub and Blackboard.\
\
Additionally, if you want to re-classify the included dog/food images or classify another set of images, you may have to 
\i restart your python kernel
\i0 . I could figure what went wrong about pre-defined values unable to be overwritten through the error report (provided below), but I could not piece together a workaround.\
\
ValueError: Variable conv1/weights already exists, disallowed. Did you mean to set reuse=True or reuse=tf.AUTO_REUSE in VarScope? Originally defined at:\
\
  File "alexnet.py", line 144, in conv\
    num_filters])\
  File "alexnet.py", line 61, in create\
    conv1 = conv(self.X, 11, 11, 96, 4, 4, padding='VALID', name='conv1')\
  File "alexnet.py", line 56, in __init__\
    self.create()}